app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.elves.druid import Druid
+from app.players.elves.elf_ranger import ElfRanger
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total_rating = 0
+    for player in players:
+        total_rating += player.get_rating()
+    return total_rating
+
+
+def elves_concert(elves: list[Druid, ElfRanger]) -> None:
```

Comment:
If we have more than 2 child classes? You will list of all? What about say parent class(not Player).
The same with feast_of_the_dwarves